ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/InternalValidationResult.java

```diff
@@ -48,6 +46,26 @@ public static InternalValidationResult create(
     return new InternalValidationResult(validationResultCode, Optional.of(description));
   }
 
+  public static InternalValidationResult reject(final String description) {
+    return create(ValidationResultCode.REJECT, description);
+  }
+
+  @FormatMethod
+  public static InternalValidationResult reject(
+      final String descriptionTemplate, final Object... args) {
+    return create(ValidationResultCode.REJECT, String.format(descriptionTemplate, args));
+  }
+
+  public static InternalValidationResult ignore(final String description) {
+    return create(ValidationResultCode.IGNORE, description);
+  }
```

Comment:
with the other 2 as format methods, might be worth this one being a format so parameters can be passed consistently

Reply:
There's two versions, one that takes a single `String` and doesn't incur the cost of `String.format` and one that takes args and does call `String.format`.

Reply:
hmm, actually I can see the potential issue here, will remove the single string versions so we get compiler errors if you specify a `%s` but don't pass an actual value.